src/messages/formatters.ts

```diff
@@ -79,6 +81,66 @@ export function blockQuote<C extends string>(content: C): `>>> ${C}` {
 	return `>>> ${content}`;
 }
 
+/**
+ * Formats the URL into `<>`, which stops it from embedding.
+ * @param url The URL to wrap.
+ * @returns The formatted content.
+ */
+export function maskedLink<C extends string>(url: C): `<${C}>`;
+
+/**
+ * Formats the URL into `<>`, which stops it from embedding.
+ * @param url The URL to wrap.
+ * @returns The formatted content.
+ */
+export function maskedLink(url: URL): `<${string}>`;
+export function maskedLink(url: string | URL) {
+	// eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+	return `<${url}>`;
+}
+
+/**
+ * Formats the content and the URL into a masked URL.
+ * @param content The content to display.
+ * @param url The URL the content links to.
+ * @returns The formatted content.
+ */
+export function hyperlink<C extends string>(content: C, url: URL): `[${C}](${string})`;
+
+/**
+ * Formats the content and the URL into a masked URL.
+ * @param content The content to display.
+ * @param url The URL the content links to.
+ * @returns The formatted content.
+ */
+export function hyperlink<C extends string, U extends string>(content: C, url: U): `[${C}](${U})`;
+
+/**
+ * Formats the content and the URL into a masked URL.
+ * @param content The content to display.
+ * @param url The URL the content links to.
+ * @param title The title shown when hovering on the masked link.
+ * @returns The formatted content.
+ */
+export function hyperlink<C extends string, T extends string>(content: C, url: URL, title: T): `[${C}](${string} ${T})`;
```

Comment:
This will not work
![Screenshot_20210701-153257](https://user-images.githubusercontent.com/73136330/124133057-bd824680-da81-11eb-8927-3b6379e3f7ca.jpg)

```suggestion
export function hyperlink<C extends string, T extends string>(content: C, url: URL, title: T): `[${C}](${string} "${T}")`;
```